markup.go

```diff
@@ -170,7 +170,19 @@ func If(cond bool, markup ...MarkupOrComponentOrHTML) MarkupOrComponentOrHTML {
 	if cond {
 		return List(markup)
 	}
-	return nil
+	var result []MarkupOrComponentOrHTML
+	for _, m := range markup {
+		if m == nil {
+			continue
```

Comment:
I think this `continue` here would cause a bug where this would produce one less element to end up in `result`, no?

```
If(false, Tag("p"), nil, Tag("p")) // produces -> [Tag("noscript"), Tag("noscript")]
```

Instead of the three noscript elements it would produce when called with `true`:

```
If(true, Tag("p"), nil, Tag("p")) // produces -> [Tag("noscript"), nil, Tag("noscript")]
```